packages/fiori-freestyle-writer/templates/listdetail/add/webapp/controller/Detail.controller.ts

```diff
@@ -139,7 +139,7 @@ export default class Detail extends BaseController {
         const originalViewBusyDelay = this.getView()!.getBusyIndicatorDelay();
         const viewModel = this.getModel<JSONModel>("detailView");<%if (template.settings.lineItem.name) {%>
         const lineItemTable = this.byId("lineItemsList") as Table;
-        const originalLineItemTableBusyDelay = lineItemTable.getBusyIndicatorDelay();<%}%>;
+        const originalLineItemTableBusyDelay = lineItemTable.getBusyIndicatorDelay();<%};%>
```

Comment:
Same as before

Reply:
removed in https://github.com/SAP/open-ux-tools/pull/984/commits/7cab7e4ad60e11fafb0b7e4a8080f2112c27ca4e